rules-reviewed/eap6/java-ee/seam/seam-ui.windup.xml

```diff
@@ -17,26 +17,60 @@
     </metadata>
 
     <rules>
-        <rule id="seam-ui-00000">
+        <rule id="seam-ui-jsf-00001">
             <when>
-                <xmlfile matches="//*[namespace-uri()='http://jboss.com/products/seam/taglib'] | //*[namespace-uri()='http://jboss.org/schema/seam/taglib']">
+                <xmlfile matches="//*[namespace-uri()='http://jboss.com/products/seam/taglib']">
                 </xmlfile>
             </when>
             <perform>
-                <classification issue-display-mode="all" title="JSF Seam Template" effort="0" category-id="mandatory">
+                <classification issue-display-mode="all" title="Seam 2.x Tag library" effort="1" category-id="mandatory">
```

Comment:
added effort as that is not really 0, you need to review and configure JSF 1.2 and add jboss-deployment-structure.xml on EAP6 or plan migration